docassemble/ALWeaver/data/questions/assembly_line.yml

```diff
@@ -2118,14 +2127,17 @@ code: |
 #----------------------------------------------------
 ---
 # Set default values if no template file to upload
+scan for variables: False
+sets:
+  - no_template_default_values
```

Comment:
I think you can use the [`only sets` feature](https://docassemble.org/docs/modifiers.html#only%20sets) here: 
```suggestion
only sets: no_template_default_values
```
TBH I didn't know about `only sets` before and I really like it. I guess the only downside is additional complication, but it seems like a really good idea for more consistent "APIs" for triggering blocks.

Reply:
I wasn't aware of this one--I like it.